src/components/my/InformationMenu.tsx

```diff
@@ -0,0 +1,59 @@
+import { ReactNode } from 'react';
+import { css, Theme } from '@emotion/react';
+
+import { textEllipsisCss } from '~/styles/utils';
+
+export type MyInformationAlignType = 'top' | 'bottom';
+
+export interface MyInformationProps {
+  label: string;
+  description: string;
+  rightElement?: ReactNode;
+  /**
+   * rightElement가 flex-start || flex-end 위치할 수 있도록하기 위해 사용됩니다.
+   */
+  align?: MyInformationAlignType;
+}
+
+export default function MyInformationMenu({
+  label,
+  description,
+  rightElement,
+  align = 'top',
+}: MyInformationProps) {
+  return (
+    <li css={wrapperCss(align)}>
+      <div css={informationCss}>
+        <span css={labelCss}>{label}</span>
+        <span css={descriptionCss}>{description}</span>
+      </div>
+      {rightElement && rightElement}
```

Comment:
LGTM bot에서 이런 경우를 경고 처리하더라구요!! 
```suggestion
      {rightElement && <>{rightElement}</>}
```

동작에는 문제없겠지만, 혹시 경고가 발생한다면 이렇게 처리하면 되는 점 공유드려욧!!

Reply:
적용완료했습니다! ede0a65